sample/src/js/connectivity-ui.js

```diff
@@ -75,6 +75,7 @@ function convertFailedTestsToString(failedTests) {
 export function displayTestQualityResults(error, results) {
   var statusContainerEl = document.getElementById('quality_status_container');
   var statusEl = statusContainerEl.querySelector('p');
+  var statusIconEl = statusContainerEl.querySelector('img');
```

Comment:
@sebakerckhof, can you please remove line 106 (which reassigns this value to `statusIconEl`)?

Reply:
And thanks for your PRs. I am reviewing the others.